modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -0,0 +1,251 @@
+This can be done on any Neo4j database.
+
+[NOTE]
+Selecting a default Cypher version on a database requires the `SET DEFAULT LANGUAGE` privilege.
+
+Selecting `CYPHER 5` as the default database language ensures that every query run on that database uses the language as it existed at the time of the Neo4j 5.26 long-term support release.
+Any changes introduced in Neo4j 2025.03 or later will not affect the semantics of the query.
+
+.Create a database with Cypher 5 as the default language
+[source, cypher]
+----
+CREATE DATABASE my_database DEFAULT LANGUAGE CYPHER 5
+----
+
+Selecting `CYPHER 25` ensures that the query will be executed using the language as it exists in the version of Neo4j that the database is currently running, provided it is on Neo4j 2025.03 or later.
+
+.Create a database with Cypher 25 as the default language
+[source, cypher]
+----
+CREATE DATABASE my_new_database DEFAULT LANGUAGE CYPHER 25
+----
+
+[[alter-default-cypher-version]]
+== Alter the default Cypher version for a database
+
+To alter the default Cypher version on a database, add `SET DEFAULT LANGUAGE <language version>` to the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/alter-databases/[`ALTER DATABASE`] command.
+
+[NOTE]
+Selecting a default Cypher version on a database requires the `SET DEFAULT LANGUAGE` privilege.
+
+.Set Cypher 5 as the default language on an existing database
+[source, cypher]
+----
+ALTER DATABASE my_new_database SET DEFAULT LANGUAGE CYPHER 5
+----
+
+.Set Cypher 25 as the default language on an existing database
+[source, cypher]
+----
+ALTER DATABASE my_database SET DEFAULT LANGUAGE CYPHER 25
+----
+
+[[show-cypher-versions]]
+== Show the Cypher version of databases
+
+The Cypher version of a database can be seen using the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/listing-databases/[`SHOW DATABASES`] command as follows:
+
+.Set Cypher 25 as the default language on an existing database
+[source, cypher]
+----
+SHOW DATABASES YIELD name, defaultLanguage
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| name | defaultLanguage 
+
+| "my_database" | "CYPHER 25"
+| "my_new_database"  | "CYPHER 5"
+| "system"           | "CYPHER 25"
+
+2+d|Rows: 3
+|===
+
+[[config-database-upgrades]]
+== Cypher versions, configuration settings, and system upgrades
+
+Changing the Cypher version of a database can also be done with the setting link:{neo4j-docs-base-uri}/operations-manual/current/configuration/configuration-settings/#config_db.query.default_language[`config_db.query.default_language`].
+This setting determines the default language for new databases where it has not been specified as part of a `CREATE` or `ALTER` database command.
+For example, `config_db.query.default_language=CYPHER_25` will set Cypher 25 as the default language for a system.
```

Comment:
can `for a system` risk confusion around if it's related to the system db or the DBMS as a whole?